net/zerotier/src/opnsense/service/templates/OPNsense/zerotier/zerotier

```diff
@@ -1,11 +1 @@
-{% set networks = [] %}
-{% for network in helpers.toList('OPNsense.zerotier.networks.network') %}
-{%   if network.enabled == '1' %}
-{%     do networks.append(network) %}
-{%   endif %}
-{% endfor %}
-{% if networks|length > 0 %}
-zerotier_enable="YES"
-{% else %}
-zerotier_enable="NO"
-{% endif %}
+zerotier_enable="{% if helpers.exists('OPNsense.zerotier.enabled') and OPNsense.zerotier.enabled == '1' %}YES{% else %}NO{% endif %}"
```

Comment:
it's harder to read/debug this, the multi-line split with

zerotier_enable="YES"
 {% else %}
 zerotier_enable="NO"
 {% endif %}

it the next option needs to be passed it'll be split right up again

Reply:
done, will commit shortly.